src/content/posts/How-to-test-Angor/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "How to Test Angor: A Step-by-Step Guide"
+published: 2024-05-18
+description: "Angor is a decentralized crowdfunding project leveraging the security of Bitcoin and the transparency of Nostr."
+image: './test-angor.svg'
+tags: 
```

Comment:
Please use Array : 

tags: ['Decentralized Crowdfunding', 'Bitcoin' ,'Nostr','Angor']

also for category